Makefile

```diff
@@ -163,7 +153,11 @@ ifneq ($(HAS_WEBAPP),)
 endif
 	rm -fr build/bin/
 
-# generate store mocks
+## Generate store mocks
 .PHONY: store-mocks
 store-mocks:
 	mockery -name=".Store" -dir server/store -output server/store/mockstore/mocks -note 'Regenerate this file using `make store-mocks`.'
+
+# Help documentatin à la https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+help:
+	@cat Makefile | grep -v '\.PHONY' |  grep -v '\help:' | grep -B1 -E '^[a-zA-Z_.-]+:.*' | sed -e "s/:.*//" | sed -e "s/^## //" |  grep -v '\-\-' | tac | awk 'NR%2{printf "\033[36m%-30s\033[0m",$$0;next;}1' | sort
```

Comment:
`tac` is not default command for macOS, so `make help` may fail in macOS.  https://unix.stackexchange.com/a/114042
I think that `perl -e 'print reverse <>' ` is more general. (But `perl` is not POSIX, so my suggestion is not perfect)

What do you think about supports of macOS?

Reply:
I would like to stay consistent with https://github.com/mattermost/mattermost-plugin-sample.

0/5 about the out of the box support of macOS. But also 0/5 on the POSIX compatibility. What do you think?

Reply:
Got it. Now, I'll merge this PR, because `make help` is not critical for development.

---

But I hope that plugin dev tools has compatibility with macOS, because I mainly use macOS for development of mattermost plugin.

In [this comments](https://github.com/mattermost/mattermost-plugin-sample/pull/19#issuecomment-436811575), lieut-data  said that he can replace `tac` with `awk`, and `awk` is POSIX command. I will ask him replacing `tac` with `awk`.